### PR TITLE
Fix for directory filter as well as some improvements

### LINKED ISSUE #145: Directory drop-down from web interface
**Describe the bug**
In version 6.2 I could select a directory to filter out my local library. When I updated to version 6.3 the directory selection is greyed out and can't select the directory I want. I tried to recreate the database, but I can't still filter by directory.

**Affected version**
6.3 d288bfa658467f3de617b8be13ae6b2b5f4ede22

**Expected behavior**
Be able to filter by directory.

**Screenshots**
image

**Desktop (please complete the following information):**
 - Manjaro Linux
 - Firefox
 - 76.0

### PR DESCRIPTION
**Pull request by TylerVigario**
- Remove the outer card from library browser area
- Migrate rest of FontAwesome classes to 5.x compatible
- Add missing enable logic for the directory filter (fixes #145)
- Update Bootswatch Darkly theme to latest
- Add enable logic to fast forward button

### CODE REVIEW

**Comment by TerryGeng** [comment]
Ready for merging? ;)

**Comment by TylerVigario** [comment]
@TerryGeng Yes sir! 😅

File: static/js/custom.js [hunk 1]
```diff
@@ -200,7 +199,7 @@ function updatePlaylist() {
                 200: displayPlaylist,
                 204: function () {
                     playlist_loading.hide();
-                    playlist_empty.show();
+                    playlist_empty.removeClass('d-none');
```
Comment (TerryGeng) [inline static/js/custom.js:202]: This is more like a choice between jquery and bootstrap, and from your code, you are the bootstrap guy :)

File: static/js/custom.js [hunk 2]
```diff
@@ -257,9 +256,11 @@ function bindPlaylistEvent() {
 
 function updateControls(empty, play, mode, volume) {
     if (empty) {
-        playPauseBtn.prop("disabled", true);
+        playPauseBtn.prop('disabled', true);
+        fastForwardBtn.prop('disabled', true);
```
Comment (TerryGeng) [inline static/js/custom.js:260]: Thanks for fixing that for me!

**Comment by TerryGeng** [comment]
And, just FYI, @azlux and I are on #mumble of IRC channel Freenode. If you'd like to have more discussion with us plz feel free to join!

File: static/js/custom.js (hunk 1 above)
Comment (TylerVigario) [reply static/js/custom.js:202]: The thinking here is to prepare for a future without jQuery, too bad Bootstrap requires it.

File: static/js/custom.js (hunk 2 above)
Comment (TylerVigario) [reply static/js/custom.js:260]: You're welcome! 😀

**Comment by TylerVigario** [comment]
> Can you remove pymumble?
 
My apologies, I had not noticed it.

> @azlux and I are on #mumble of IRC channel Freenode

I'll have to stop by sometime.

**Pull request merged**